enhancements/monitoring/alerting-consistency.md

```diff
@@ -0,0 +1,195 @@
+---
+title: alerting-consistency
+authors:
+  - "@michaelgugino"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-03
+last-updated: 2021-02-03
+status: implementable
+---
+
+# Alerting Consistency
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Alerts are a mess.  There's not really formal guidelines to follow when
+implementing alerts for a given component.  These are the guidelines.
+
+## Motivation
+
+Improve Signal to noise of alerts.  Align alert levels "Critical, Warning, Info"
+with operational expectations to allow easier triaging.  Ensure current and
+future alerts have documentation covering problem/investigation/solution steps.
+
+### Goals
+
+* Define clear criteria for designating an alert 'critical'
+* Define clear criteria for designating an alert 'warning'
```

Comment:
Do we only care about those 2 levels, or is the goal to describe criteria for all of the levels?

Reply:
You tell me.  I just wanted to get the conversation started.  I would prefer to have 3 alert levels, "Critical" "Major" and "Minor".  Since that ship might have already sailed, I am focusing on what we have today.

Reply:
Other levels are mentioned in the text below, so I wasn't sure if you were just focusing on the top 2 as a start or if there was another reason they weren't included here. I'd be OK with starting by nailing down the criteria for critical and warning and seeing where that takes us.